{'histories': array([0, 0, 1, ..., 0, 0, 1]), 'resize_history': 40, 'vocab_size': 10, 'compress_context_size': 20, 'compress_context_count': 2, 'episode_size': 100}
!!!! 10
number of parameters: 25.44M
running on device cpu
[34m[1mwandb[39m[22m: [33mWARNING[39m Calling wandb.run.save without any arguments is deprecated.Changes to attributes are automatically persisted.
/home/aaderevyagin/airi_final/venv/lib/python3.10/site-packages/torch/cuda/__init__.py:619: UserWarning: Can't initialize NVML
  warnings.warn("Can't initialize NVML")
  0%|                                                                                                                                          | 0/2000 [00:00<?, ?it/s]
Traceback (most recent call last):
  File "/home/aaderevyagin/airi_final/airi-screening-part-1/./algorithms/in_context_learning/AD.py", line 316, in <module>
    train()
  File "/home/aaderevyagin/airi_final/venv/lib/python3.10/site-packages/pyrallis/argparsing.py", line 158, in wrapper_inner
    response = fn(cfg, *args, **kwargs)
  File "/home/aaderevyagin/airi_final/airi-screening-part-1/./algorithms/in_context_learning/AD.py", line 299, in train
    trainer.run()
  File "/home/aaderevyagin/airi_final/airi-screening-part-1/algorithms/in_context_learning/trainer.py", line 88, in run
    batch = next(data_iter)
  File "/home/aaderevyagin/airi_final/venv/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 631, in __next__
    data = self._next_data()
  File "/home/aaderevyagin/airi_final/venv/lib/python3.10/site-packages/torch/utils/data/dataloader.py", line 675, in _next_data
    data = self._dataset_fetcher.fetch(index)  # may raise StopIteration
  File "/home/aaderevyagin/airi_final/venv/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in fetch
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/aaderevyagin/airi_final/venv/lib/python3.10/site-packages/torch/utils/data/_utils/fetch.py", line 51, in <listcomp>
    data = [self.dataset[idx] for idx in possibly_batched_index]
  File "/home/aaderevyagin/airi_final/airi-screening-part-1/utils/dataset_utils.py", line 173, in __getitem__
    result += [self.config.vocab_size for j in range(3 * self.compress_context_size)]
  File "/home/aaderevyagin/airi_final/airi-screening-part-1/utils/dataset_utils.py", line 173, in <listcomp>
    result += [self.config.vocab_size for j in range(3 * self.compress_context_size)]
AttributeError: 'HistoriesDataset' object has no attribute 'config'